controller_interface/include/controller_interface/controller_interface.hpp

```diff
@@ -43,28 +43,24 @@ class ControllerInterface : public rclcpp_lifecycle::node_interfaces::LifecycleN
   ~ControllerInterface() = default;
 
   CONTROLLER_INTERFACE_PUBLIC
-  controller_interface_ret_t
   virtual
+  controller_interface_ret_t
   init(
     std::weak_ptr<hardware_interface::RobotHardware> robot_hardware,
     const std::string & controller_name);
 
-  CONTROLLER_INTERFACE_PUBLIC
-  std::shared_ptr<rclcpp_lifecycle::LifecycleNode>
-  get_lifecycle_node();
-
   CONTROLLER_INTERFACE_PUBLIC
   virtual
   controller_interface_ret_t
   update() = 0;
 
+  CONTROLLER_INTERFACE_PUBLIC
```

Comment:
Just wondering if these moves are relevant changes?

Reply:
just for style.